File: 40130112056/Remove_Duplicates.py
def RemoveDuplicates(Array):
    lenOfArray=0
    for i in range(0 , len(Array)):
        if(Array[i]!= Array[i-1]):
            lenOfArray +=1
    return lenOfArray         

array = [1 , 1, 2 , 2 , 3, 4 , 5, 5]
print(RemoveDuplicates(array))

File: 40130112106/removeDuplicates.js
function removeDuplicates(array) {
  let lenOfArray = 0;
  for (let i = 0; i < array.length; i++) {
    if (array[i] !== array[i - 1]) {
      lenOfArray++;
    }
  }
  return lenOfArray;
}

let array = [1, 2, 2, 3, 3, 3, 4, 5];
console.log(removeDuplicates(array));
